### DISCUSSION TITLE
Support OR function (regex)

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1890] [Anything Else]
"RegEx in regards to the OR function. So lets say Page -> Contain -> bread OR pate (bread|pate)"

(request via email)

**Update**: You can now filter your stats by multiple countries, pages, sources and more, all at once. Click on the "Filter" button in your dashboard to start exploring.

**Comment by tiacop** [comment]
Would be super useful - also for goals! 

Specially for localized pages with translated slugs like this right now it's a hell:

/en/institutes-centers/institute-for-renewable-energy
/de/institutes-centers/institut-fuer-erneuerbare-energie
/it/institutes-centers/istituto-per-le-energie-rinnovabili

I don't like having to add them up manually :wink:

**Reply by metmarkosaric** [reply]
hi @tiacop! you should be able to do something like that already with the current setup. something like `/*/institutes-centers/**` should track all of these in one group. see https://plausible.io/docs/pageview-goals#pageview-goals-support-wildcards for more details

**Reply by tiacop** [reply]
hi @metmarkosaric,
we use wildcards already and it works great. But the specific usecase is to provide data for 15 institutes with translated slug. 

/en/institutes-centers/institute-for-renewable-energy
/de/institutes-centers/institut-fuer-erneuerbare-energie
/it/institutes-centers/istituto-per-le-energie-rinnovabili

/en/institutes-centers/institute-for-biomedicine
/de/institutes-centers/institut-fuer-biomedizin
/it/institutes-centers/istituto-di-biomedicina

/en/institutes-centers/institute-for-mummy-studies
/de/institutes-centers/institut-fuer-mumienforschung
/it/institutes-centers/istituto-per-lo-studio-delle-mummie

... +12 more

Having goals like `**institute-for-mummy-studies** OR **institut-fuer-mumienforschung** OR **istituto-per-lo-studio-delle-mummie**` would simplify work a lot in order to group the translated pages form the same institute.

Thanks for your great tool!

**Reply by metmarkosaric** [reply]
ah yes, i see. to group the pages by the second part of the URL only is not possible at the moment. one part of the URL needs to be consistent for the current wildcards to work so yeah adding the ability to use `OR` would be perfect for your use case

**Comment by tiacop** [comment]
hey @metmarkosaric, any news on this feature request? 
It is still very annoying to sum up the totals for related and translated pages manually ... specially if you sit in front of such a nice tool that should handle regex with ease 😉

**Reply by metmarkosaric** [reply]
sorry @tiacop but i don't have any news on this! it would be useful feature to add for sure and i will make sure to post here as soon as we have some updates on it! thank you!

**Comment by metmarkosaric** [comment]
**Update**: You can now filter your stats by multiple countries, pages, sources and more, all at once. Click on the "Filter" button in your dashboard to start exploring.

**Reply by tiacop** [reply]
Hey @metmarkosaric, the functionality is great and super-usefull, thanks a lot!
The UI is not perfect, if you try to enter manually a few keywords (ex. `/**keyword1**` `/**keyword2**`) not always the suggestion "Filter by" appears and if you hit enter in this case you loose your input.

With suggestion "Filter by" and the possibility to hit enter
<img width="416" alt="image" src="https://user-images.githubusercontent.com/61466670/228277372-3224967d-f394-4ea7-9741-2883c2288470.png">

No "Filter by" suggestion this time ... you better not hit enter now or you loose your input ;)
<img width="413" alt="image" src="https://user-images.githubusercontent.com/61466670/228278827-aa1c4e10-5c05-48b9-9d4f-76205a7dbac4.png">

**Reply by metmarkosaric** [reply]
thanks for the feedback @tiacop! 

we're already working on a fix here https://github.com/plausible/analytics/pull/2797

btw when you use "contains", you don't need to type any of the wildcards/`**`. you can just type the keyword itself

**Reply by metmarkosaric** [reply]
hi @tiacop! we've deployed the fix now so should work as expected even when you hit enter. just do enter straight up keywords without the asterisk as it's not necessary when using "contains"
